src/PathItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PathItem = ({pathName, pathItem}) => {
```

Comment:
All about the pure functional components when we can use them. You can actually remove the curly braces and the explicit `return` and parens here too 👍